views/_includes/layouts/base.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
```

Comment:
Can the page title be dynamically added to the template via the controller?

Reply:
Yes!